Examples/PairwiseDocNewUIESA/example.tsv

```diff
@@ -0,0 +1,100 @@
+1	EuGH-Urteil stärkt Eltern	This is a fake reference.	ECJ ruling strengthens parents	ECJ ruling strengthens parents
```

Comment:
Is this new file or taken from another example? If re-used, please don't add it to the repo but modify the bash scripts to refer to the one from the other directory.

Reply:
Modified the bash script and deleted this file.